src/data.js

```diff
@@ -5452,6 +5452,33 @@ module.exports = [
     ],
   },
 
+  {
+    name: 'Trezy',
+    description: 'Software engineer, JavaScript livestreamer, and a wannabe cyborg.',
+    url: 'https://trezy.com/uses',
```

Comment:
This page 404s

Reply:
It shouldn't be:
![Screen Shot 2020-02-21 at 12 36 06 PM](https://user-images.githubusercontent.com/442980/75061661-c8e55b00-54a6-11ea-9e2d-4a958a3b9882.png)

What browser/OS are you using? Are there any errors in the console?

Reply:
Might have been a weird glitch